cmd/repo-updater/repoupdater/server.go

```diff
@@ -203,7 +203,7 @@ func (s *Server) handleExternalServiceSync(w http.ResponseWriter, r *http.Reques
 	if sourcer = s.Sourcer; sourcer == nil {
 		db := database.NewDBWith(logger, s)
 		depsSvc := livedependencies.GetService(db, nil)
-		sourcer = repos.NewSourcer(db, httpcli.ExternalClientFactory, repos.WithDependenciesService(depsSvc))
+		sourcer = repos.NewSourcer(s.Logger.Scoped("handleExternalServiceSync", ""), db, httpcli.ExternalClientFactory, repos.WithDependenciesService(depsSvc))
```

Comment:
same as the other suggestions - please audit scopes, scopes should apply to what is going to get logged

```suggestion
		sourcer = repos.NewSourcer(s.Logger.Scoped("repos.Sourcer", ""), db, httpcli.ExternalClientFactory, repos.WithDependenciesService(depsSvc))
```